flags.go

```diff
@@ -285,6 +292,7 @@ type flagStorage struct {
 	RenameDirLimit int64
 
 	// GCS
+	Url                                *url.URL
```

Comment:
The fields in this struct match the names of the flags. Should we change `Url` to `Endpoint` instead, to match the new flag `--endpoint`?

Reply:
Done.